static/app/utils/analytics.tsx

```diff
@@ -34,18 +34,6 @@ import {Hooks} from 'sentry/types/hooks';
 export const trackAnalyticsEventV2: Hooks['analytics:track-event-v2'] = (data, options) =>
   HookStore.get('analytics:track-event-v2').forEach(cb => cb(data, options));
 
-/**
- * @deprecated Use trackAdvancedAnalyticsEvent or another function generated by makeAnalyticsFunction
- */
-export const trackAnalyticsEvent: Hooks['analytics:track-event'] = options =>
-  HookStore.get('analytics:track-event').forEach(cb => cb(options));
-
-/**
- * @deprecated Use trackAdvancedAnalyticsEvent or another function generated by makeAnalyticsFunction
- */
-export const trackAdhocEvent: Hooks['analytics:track-adhoc-event'] = options =>
-  HookStore.get('analytics:track-adhoc-event').forEach(cb => cb(options));
```

Comment:
Can we also remove the hook types?

Reply:
@evanpurkhiser I'm going to do that in the next PR when I remove the actual hooks. I just want to do this first.

Reply:
👍